src/main/java/org/craftercms/studio/impl/v1/service/deployment/PublishingManagerImpl.java

```diff
@@ -215,7 +216,7 @@ else if (StringUtils.equalsIgnoreCase(LIVE_ENVIRONMENT, item.getEnvironment()) |
                     LOGGER.debug("Environment is live, transition item to LIVE state {0}:{1}", site, path);
 
                     // check if commit id from workflow and from object state match
-                    if (itemMetadata.getCommitId() != null && itemMetadata.getCommitId().equals(item.getCommitId())) {
+                    if (Objects.isNull(itemMetadata.getCommitId()) || itemMetadata.getCommitId().equals(item.getCommitId())) {
```

Comment:
What does `itemMetadata.getCommitId().equals(item.getCommitId())` mean?

Reply:
that is the foundation for multiple publish requests of the same file (multiple versions, it will have status "LIVE" only if the version being published is the one in item metadata (the latest version)